test/ssl-tests/32-compressed-certificate.cnf.in

```diff
@@ -0,0 +1,139 @@
+# -*- mode: perl; -*-
+# Copyright 2016-2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+## SSL test configurations
+
+package ssltests;
+
+use OpenSSL::Test::Utils;
+
+our @tests = ();
+
+our @tests = (
+    {
+        name => "no-compressed-certificates",
+        server => {
+          "Options" => "-TxCertificateCompression,-RxCertificateCompression",
+        },
+        client => {
+          "Options" => "-TxCertificateCompression,-RxCertificateCompression",
+          "MinProtocol" => "TLSv1.3",
+        },
+        test   => {
+          "ExpectedResult" => "Success",
+        },
+    },
+    {
+        name => "server-compressed-certificates",
+        server => {
+          "Options" => "TxCertificateCompression,RxCertificateCompression",
```

Comment:
Should there be a change in ssl_test_new to actually precompress certificates? Otherwise the TxCertificateCompression option will never actually send compressed certificates?